src/blocks/button.css

```diff
@@ -0,0 +1,22 @@
+/* Link */
+.product-card__button {
+  box-sizing: border-box;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 38px;
+  text-align: center;
+  text-decoration: none;
+  color: #fff;
+  border: 1px solid #00acdc;
+  background-color: #00acdc;
+  border-radius: 5px;
+  margin: auto;
+  width: 166px;
+  height: 40px;
+}
+
+.product-card__button:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border: 1px solid #00acdc;
+}
```

Comment:
In this task you have only 2 separate blocks, so you need only 2 files with styles